### PR TITLE
docs(v2): Add new showcase user

### PR DESCRIPTION
**Pull request by JeremyDolle**

### CODE REVIEW

File: website/src/data/users.js [hunk 1]
```diff
@@ -559,6 +559,14 @@ const users = [
     source: 'https://github.com/facebook/react-native-website',
     tags: ['facebook', 'large', 'highlight', 'design', 'versioning'],
   },
+  {
+    title: 'React Native Boilerplate',
+    description: 'A React Native project template for building solid applications through separation of concerns between the UI, state management and business logic.',
+    preview: require('./showcase/reactnativeboilerplate.png'),
+    website: 'https://thecodingmachine.github.io/react-native-boilerplate/',
+    source: 'https://github.com/thecodingmachine/react-native-boilerplate',
+    tags: ['thecodingmachine', 'redux', 'react-native', 'architecture', 'fastlane', 'boilerplates', 'react-native-boilerplate', 'redux-toolkit'],
```
Comment (slorber) [inline website/src/data/users.js:568]:
```suggestion
    tags: [],
```

removing tags, we assign tags ;)

**Comment by slorber** [comment]
thanks, lgtm :) 

didn't know this project, is there a blog post for the v3 or something?

**Comment by JeremyDolle** [comment]
> thanks, lgtm :)
> 
> didn't know this project, is there a blog post for the v3 or something?

I just make a [reddit post](https://www.reddit.com/r/reactnative/comments/mtwtj0/update_about_our_light_simple_and_scalable_react/) for now. Do you think I should detail the V3 release in a blog post ? What kind of information do you want to find about a release blog post ? I'm interested 😀

**Comment by slorber** [comment]
@JeremyDolle I run a french React newsletter (https://reacthebdo.fr/) and Twitter news threads (https://twitter.com/sebastienlorber/status/1384171018831335432) and would be happy to include it if there was a more detailed post presenting the project. 

For each major you should present the current state, what's new etc, that helps marked the project. I basically want a TLDR of what your project is about, and how it is different from alternatives like Ignite etc (without having to read your whole doc).

You can add a blog to the docusaurus site ;)

**Comment by JeremyDolle** [comment]
@slorber Yes I am French and I follow your newsletter 🇫🇷 😉  !! So I will do a blog post to detail what's in the box and send you the url on discord or twitter as you prefer ! See you soon 👋

**Comment by slorber** [comment]
thanks :)

**Pull request merged**